doc/example/bayes_factors.ipynb

```diff
@@ -0,0 +1,1427 @@
KhRA0+fPuUf40qzjB49Gjdv3kSHDh1gamrKnx//xNatW6GlpQWBQIAxY8bg0aNH/LKzZ8+iQoUKfNBdmPzHU25urljdwZSUFJw7dw6DBw+GgoICnJ2dJRa4578BUq9ePQQFBSElJQUTJkyAQCBAhQoV4OPjg//++++72/jw4YNYTcRly5bx5YwAwN/fH5UqVYK+vj4UFBTg7u6OY8eOFc8OMQzDML+EBYKlyNSpU9G+fXvcv39f7PG4uDgEBwdDX1+f/cCWoDZt2kBbWxvVq1eHmpoa9u3bV2AdSSbs+baXq02bNqhQoQKaN2+O8+fP80lSuPV+ZS6jtNdLHDlyJEJDQ/mhqW/evMGIESNQq1YtmJubY/DgwXxiDmmdp/S9Atxv3ryBt7c3BAIBjIyMULduXdSvX5+v1ybJ/WnXrh0UFBT4bLT5paamYsWKFfzohaIG3yKRCNnZ2XB2doaGhgY2btzIL3v8+DGUlJRw5coVHDhwAPr6+oXenPhdkydPRsWKFWFsbIyAgAAMGjQIbdu25RMr/ewYX7hwIdq0aYNGjRqhZ8+eYomT3r17x998ASRzM4I7phYtWoTGjRsDyKtPy41kCA8Ph0AggEAg4Id8/2g7sbGxUFNTQ7NmzbBt2zZ++YcPH7Br1y5cvXqV3axkGIaRIiwQLCUuXryI2rVro2LFipg7d26B5RkZGUXqyWH+jrVr16Ju3bqIiopCREQEevXqherVq8PNzY2fqyRJ+TMpbtiwAf/99x/GjRuHvn37okmTJpCTk8PQoUNx584d/u69paUlwsLCfrptoVCIrKws+Pn5QVFRkU8GIxQK8eXLF1haWhaayKi4cRfSDx48gJOTE5SVleHl5YXdu3fz64hEIolnMS0q7jPctWsXBg0aBGNjY/j6+uLKlSvIyclBfHw85s6di8jISL7HRhLBRP65gXPmzEH37t0hLy+Pbt26/bAnqai4mylJSUkYPHgwdHV1YWhoiDZt2kBHRwetW7cGkNfzW79+/d/6fPPPDUxLS8Pt27f5ZYmJiejSpQsfEIWHh/+w7AXX3s2bN0NHRweBgYHYvn07BAIBKlWqhDFjxhQ6h1tSw5OFQiFmzpzJz/0bPXo0WrVqBSAvWHVzc0NgYGCRPsuPHz8iPDwcffr0gYGBATp06IALFy4Ua/sZhmGY38cCwVLiy5cvOHXqFIYOHQp5eXnY29vzPQBMych/kR0eHi6WOfDdu3fYvXs3WrRogWrVqqF3794SHW7IXVT6+fmhRYsWOHr0qNjyHTt2QENDA6qqqpg4cSIGDBiAOnXq/HR735KWeon5sxYOGDAAbdq0QcuWLWFsbAwrK6tCs4NK83BQrlfvzJkzqFevHgYNGoRdu3Z9N1mHpHDv+5MnT9C4cWM8ePAAjx8/xtKlS9GiRQsYGRlh8uTJBeY0/kj+JEbh4eEYNWoUBg8ejPj4eHz+/BkxMTGYOHEiOnfujB07diAlJQUikQiOjo7o27fvb+8DkDcX0crKCk2aNIGWlpbYnMNr167B3NwcVatWxdixY3H79u0f9r7Wq1cPq1evBpCXQdPU1BShoaGQlZWFrq4uli1bVuD1JeXevXs4c+YMcnNz4enpiZCQEH5Z586dERsb+0vbe/ToEVatWgUPDw8YGBhg1KhReP369d9uNsMwDPOHWCBYynz8+BEHDhxA69atUbVqVfTr14+vacWUjKCgIJiamqJz584F6uk9efIE//zzj8TmyAH/C3COHz+OGjVq4OrVq8jIyOCXJyQkICEhAQCwYsUK6OjowMvL67up70tDvURun4cOHYqmTZvi7du3APIS90yZMgUqKiowNTVFUFDQd8tiSCMLCwt+GGJMTAw0NDSQmJgIIC/7q7QMs5s2bZpYxlKRSITLly9j8uTJsLW1hZaWFi5fvvxL2+zQoQNsbGzQv39/6Orqok6dOmJzPzmJiYmYN28eGjZs+FvzA7lgjqsxuWHDBsTExEAgEPBZmfN/f9asWYMaNWpASUnpu8HNwYMH4ebmhqysLHz8+BGqqqrYv38/AKBbt26oXbs22rZt+8ttLQndunWDo6MjXr9+jdWrV0NBQUEsGVZ+3Hv3/v17XLx4EWvXrhXr7YyLi0NISAgaNmwIOzs7qR2GzTAMU16xQFBKccOLPnz4gGPHjmH58uViFx0vXrzA2rVrYWhoCA0NDbHJ+czfxwUa6enp2LRpE0xMTCAvL49169YVWDcrK0sqync0bdoU06dP5/9OTk7Gpk2bUKNGDdSqVQva2tq4c+cOMjMzf1jmQZrrJeYPUtPT02Fvb49//vmnwHqLFy+GsrIyXF1d0a9fvwJ156TRixcv0LRpU/z7778AAHV1dSxfvhxA3nw7Ly+vQoeJlxTuuEhISMCcOXMwbty4AuukpaVh//79GDFiRJG+E9z3bO/evahZsyZ/k8vY2Jjvpbp9+7ZYT11OTg6uXLnyR8NQU1JSULt2bT5YGzhwIF9qJDU1FZMnTy6QEGrDhg0A/ncMfvjwAS9fvoRIJML9+/cxZ84cZGRkYPny5XBwcOBLLGzevBkzZszgg1pJBkdJSUl4/Pgxf2MIyEtIpK2tDYFAAENDQ8yZM+en23F0dET9+vVRs2ZNVK5cGZMnT+aD58zMTBw4cOCnmYgZhmGYkscCQSnn6OgIPT09VK9eHfLy8hg+fDifOU8oFOLmzZtsDkYJ6tWrF7Zs2YIHDx4gICAAcnJysLW1lbphugkJCbCxsRFL6T527FjY2tpi+PDhOHLkCDw9PeHs7Fyk7Ul7vUTuxsmQIUNga2tbINC7ffs2unbtisWLF6NKlSpiQ9+kVXZ2NkxNTbF161bMmzcPlpaWfDD15s0b6Ojo8MmhJDm80M3NDQKBACoqKrhw4UKhNwS4oKCow3GHDBnCB5bz58+Hnp4en31327ZtGDx48Hd7qX7HgwcPYG9vj/T0dMTHx0NRUZH/TicmJsLLy4tP6lJYgXShUIiePXti5syZf
```

Comment:
Why not steppingstone sampling instead of TI? Both show same accuracy, or steppingstone sampling looks even better. The potential bias of steppingstone sampling for small number of temperatures was not be found to be problematic according to the source you mentioned. 
I am fine with the suggestions, I was just wondering why you exclude steppingstone sampling.

Reply:
no specific reason, will include it